vars/kibanaPipeline.groovy

```diff
@@ -303,33 +303,23 @@ def doSetup() {
   }
 }
 
-def buildOss(maxWorkers = '') {
-  notifyOnError {
-    withEnv(["KBN_OPTIMIZER_MAX_WORKERS=${maxWorkers}"]) {
-      runbld("./test/scripts/jenkins_build_kibana.sh", "Build OSS/Default Kibana")
-    }
-  }
-}
-
 def getBuildArtifactBucket() {
   def dir = env.ghprbPullId ? "pr-${env.ghprbPullId}" : buildState.get('checkoutInfo').branch.replace("/", "__")
   return "gs://ci-artifacts.kibana.dev/default-build/${dir}/${buildState.get('checkoutInfo').commit}"
 }
 
-def buildXpack(maxWorkers = '', uploadArtifacts = false) {
+def buildKibana(maxWorkers = '') {
```

Comment:
Do we need to preserve `uploadArtifacts`?

Reply:
We removed the option and are always uploading the artifacts since we were previously only doing it for the default distribution.